services/galley/test/integration/API.hs

```diff
@@ -2418,6 +2421,63 @@ deleteMembersUnqualifiedFailO2O = do
   o2o <- decodeConvId <$> postO2OConv alice bob (Just "foo")
   deleteMemberUnqualified alice bob o2o !!! const 403 === statusCode
 
+putQualifiedConvRenameFailure :: TestM ()
+putQualifiedConvRenameFailure = do
+  conv <- randomId
+  qbob <- randomQualifiedUser
+  let qconv = Qualified conv (qDomain qbob)
+  putQualifiedConversationName (qUnqualified qbob) qconv "gossip"
+    !!! const 404 === statusCode
```

Comment:
Just nitpicking: how about asserting on a label too?

Reply:
Done in b0bb104f44b5a31d484c26e523fa15ef9024ee32.